chall01/alcohen.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+import string
+import sys
+
+def error():
+    print("usage: ./alcohen.py <a-zA-Z string>")
+    exit()
```

Comment:
I would suggest using sys.argv[0] for avoiding hardcoding alcohen into your print.